jetty-ee10/jetty-ee10-demos/jetty-ee10-demo-embedded/src/main/java/org/eclipse/jetty/ee10/demos/OneWebAppWithJsp.java

```diff
@@ -45,8 +45,9 @@ public static Server createServer(int port) throws FileNotFoundException
         // the webapp will unpack itself.
         WebAppContext webapp = new WebAppContext();
         webapp.setContextPath("/");
+        ResourceFactory resourceFactory = webapp.getResourceFactory();
```

Comment:
You may want to inline this variable, or rename it to `webappResourceFactory` as I think we should start being precise and cautious about where we bind the mount's lifecycle.